lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -749,6 +749,14 @@ be present in ``test`` definition.</xs:documentation>
         <xs:documentation xml:lang="en">Maximum amount of time to let test run.</xs:documentation>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="name" type="xs:string" use="optional">
```

Comment:
what do you think about `title`, that mirrors what we do for the tool tag, and would leave space for `description` should we decide to use that one day

Reply:
After talking with @bgruening about the tours that are being worked on, I was saving "title" to be used for the tour step names, while "name" would be for internal planemo testing. If you disagree, I can change it.

Reply:
Why should this be different in tours ?

Reply:
The name label would be used to show what exactly is being changed/tested, while the tour title would be in order to explain the purpose of what was being tested. I was thinking one would be more from a dev-perspective, while the other is more user-oriented.

Reply:
I supposed that using ```label``` would do just as well. I'll change to ```title```.